app/master/atom.py

```diff
@@ -16,12 +25,14 @@ def __init__(
         :type expected_time: float | None
         :type actual_time: float | None
         :type exit_code: int | None
+        :type state: `:class:AtomState` | None
```

Comment:
Since we don't use Sphinx, you should just be able to do `AtomState` here (and actually IntelliJ should detect that syntax and use it for type hinting). We don't use the `:class:` syntax anywhere else in the app.